src/core/services/theming/theming.js

```diff
@@ -689,6 +717,45 @@ function generateAllThemes($injector, $mdTheming) {
     delete palette.contrastStrongLightColors;
     delete palette.contrastDarkColors;
 
+    function getContrastType(hueName) {
+      if (defaultContrast === 'light') {
+        if (darkColors.indexOf(hueName) > -1) {
```

Comment:
Inequality check is probably faster mathematically than "greater than" check. Updated.